AcceptanceTests/muters_files_to_mutate_output.txt

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
Don't commit this file. I would've thought the `.gitignore` should have caught this, but it didn't.

Reply:
if you don't mind adding it to the `.gitignore` file, that would be awesome!